airbyte-workers/src/main/java/io/airbyte/workers/ApplicationInitializer.java

```diff
@@ -163,7 +163,7 @@ public void onApplicationEvent(final ServiceReadyEvent event) {
       log.info("Starting worker factory...");
       workerFactory.start();
 
-      log.info("Application initialized (mode = {}).", workerPlane);
+      log.info("Application initialized.");
```

Comment:
Do we see the `environment.getActiveNames` somewhere? Otherwise, I think it could be worth logging for debugging purpose.